streams/ca.m3u

```diff
@@ -169,3 +169,13 @@ https://cbcrclinear-tor.akamaized.net/hls/live/2042765/CBCRCLINEAR_TOR_11/master
 https://cbcrclinear-tor.akamaized.net/hls/live/2042756/CBCRCLINEAR_TOR_2/master5.m3u8
 #EXTINF:-1 tvg-id="CBRTDT.ca",CBC Calgary (CBRT-DT) [Geo-blocked]
 https://cbcrclinear-tor.akamaized.net/hls/live/2042763/CBCRCLINEAR_TOR_9/master5.m3u8
+#EXTINF:-1 tvg-id="ZoomerRadio.ca" status="online",ZoomerRadio
+https://ythls.onrender.com/channel/UCon2a9opDXmI-i3Be6PYBQA.m3u8
+#EXTINF:-1 tvg-id="ClassicalFM.ca" status="online",ClassicalFM
+https://ythls.onrender.com/channel/UC_DweJsPH4mvcRU3aFwVtqg.m3u8
+#EXTINF:-1 tvg-id="TVOKids.ca" status="online",TVO kids
+https://ythls.onrender.com/channel/UCGPPdIn67ZCLhhHABoQpNUw.m3u8
```

Comment:
Seems to either be Geo or not 24/7

Reply:
Just tested and TVO Kids is indeed Geo to Canada.